### PR TITLE
Fix/spacy_transformers

### LINKED ISSUE #700: [Tutorials] Issues with transformes in the SpaCy tutorial
After importing space and loading the model, this error may be displayed:

**ValueError: [E002] Can't find factory for 'transformer' for language English (en). This usually happens when spaCy calls `nlp.create_pipe` with a custom component name that's not registered on the current language class. If you're using a Transformer, make sure to install 'spacy-transformers'. If you're using a custom component, make sure you've added the decorator `@Language.component` (for function components) or `@Language.factory` (for class components).**

In order to get rid of it, the following line could be added after importing Rubrix and installing SpaCy datasets:
**%pip install spacy-transformers -f https://download.pytorch.org/whl/torch_stable.html**

There are other suggestions (i.e. **%pip install spacy-transformers**), but sometimes it fails.

### PR DESCRIPTION
**Pull request by leireropl**
Added "%pip install space-transformers" line to avoid errors when loading the model

Closes #700

### CODE REVIEW

**Comment by leireropl** [comment]
Closes #700

File: docs/tutorials/02-spacy.ipynb [hunk 1]
```diff
@@ -73,7 +73,9 @@
    },
    "outputs": [],
    "source": [
-    "%pip install datasets spacy~=3.0 protobuf -qqq"
+    "%pip install datasets spacy~=3.0 protobuf -qqq\n",
+    "%pip install spacy-transformers -f https://download.pytorch.org/whl/torch_stable.html\n",
+    "#install this or %pip install spacy-transformers to avoid problems when loading the model"
```
Comment (dcfidalgo) [inline docs/tutorials/02-spacy.ipynb]: We should explicitly install PyTorch, and then specify `spacy[transformers]` which automatically installs the spacy-transformers package:
```suggestion
    "%pip install torch datasets spacy[transformers]~=3.0 protobuf -qqq",
```

**Review by dcfidalgo — CHANGES_REQUESTED** [review]
Following Dani's comment in #702, we should enforce the installation of pytorch in the beginning, see inline comment.

**Review by dcfidalgo — APPROVED** [review]
Nice! LGTM!

**Pull request merged**